code/src/index.css

```diff
@@ -5,9 +5,182 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #000;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.movie-list {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.movie-list a{
+ position: relative;
+ text-decoration: none;
+ width: 25%;
+}
+
+.movie-list img{
+  width: 100%;
+  height: 100%;
+}
+
+.movie-info {
+  display: none;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 20px;
+  background: rgba(0,0,0,0.50);
+  color: whitesmoke;
+}
+
+.movie-list a:hover .movie-info{
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.back-link {
+  text-decoration: none;
+  color: whitesmoke;
+  font-weight: bold;
+  font-size: 20px;
+  padding-left: 40px;
+  display: flex;
+  align-items: center; 
+}
+
+.back-arrow {
+  width: 25px;
+  height: 25px;
+  margin-right: 5px;
+  transition: .4s ease-in-out;
+}
+
+.back-link:hover .back-arrow{
+  margin-right: 10px;
+}
+
+.movie-backdrop {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
```

Comment:
I notice that your background image does not cover the full view in desktop view. I struggled myself with that and ended up doing like this:
``
.details-container {
    display: flex;
    flex-direction: row;
    background-repeat: no-repeat;
    background-size: cover;
    background-position: center;
    background-color: black;
    position: fixed;
    width: 100%;
    height: 100%;
    border: none;
    align-items: flex-end;
}
``